<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeDpViewWidget</class>
 <widget class="QWidget" name="C_SdNdeDpViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="pc_VerticalLayout" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="pc_HorizontalLayout2" stretch="1,0,0">
       <item>
        <spacer name="pc_HorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubIconOnly" name="pc_BtnMinMaxDbSelector">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="pc_HorizontalSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>29</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="pc_HorizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeDpSelectorWidget" name="pc_WidgetDpDiag" native="true"/>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeDpSelectorWidget" name="pc_WidgetDpNvm" native="true"/>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeDpSelectorWidget" name="pc_WidgetDpCom" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeDpSelectorWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeDpSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
